layouts/events/list.html

```diff
@@ -124,98 +124,48 @@ <h5 class="modal-title" id="suggestionModalLabel">Events & training suggestion</
       } else {
-        var eventsHTML = `<div class="row mt-3 mb-3">`;
-        if (local.length > 0){
-          eventsHTML += `<h5>Location</h5>`;
-          local.forEach((lc) => {
-            eventsHTML += `<div class="form-check">   
-                            <input class="form-check-input" type="checkbox" id="${lc}" name="location" value="${lc.replaceAll(" ", "-").toLowerCase()}">
-                            <label class="form-check-label" for="${lc}">${(lc == "Online") ? "Online/Hybrid" : lc}</label>
-                           </div>`;
-          });
-        };
-        if (category.length > 0){
-          eventsHTML += `<h5 class="mt-3">Category</h5>`;
-          category.forEach((ct) => {
-            eventsHTML += `<div class="form-check">   
-                            <input class="form-check-input" type="checkbox" id="${ct}" name="category" value="${ct.replaceAll(" ", "-").toLowerCase()}">
-                            <label class="form-check-label" for="${ct}">${ct}</label>
-                           </div>`;
-          });
-        };
-        if (level.length > 0){
-          eventsHTML += `<h5 class="mt-3">Level</h5>`;
-          level.forEach((lv) => {
-            eventsHTML += `<div class="form-check">   
-                            <input class="form-check-input" type="checkbox" id="${lv}" name="level" value="${lv.replaceAll(" ", "-").toLowerCase()}">
-                            <label class="form-check-label" for="${lv}">${lv}</label>
-                           </div>`;
-          });
-        };
-        eventsHTML += `</div>
-                       <div class="col-lg">
-                         <div class="row g-2" id="events-container">`;
+        var eventsHTML = `<div class="row mt-3 mb-3"></div>
+                            <div class="col-lg">
+                              <div class="row g-2" id="events-container">`;
         eventList.forEach((event, ind) => {
-          var this_event_local, this_event_category, this_event_level;
           var event_date = new Date(event.banner_date);
-          if (typeof event.location !== 'undefined'){
-            this_event_local = event.location.map(lc => lc.replaceAll(" ", "-").toLowerCase()).join(" ");
-          };
-          if (typeof event.category !== 'undefined'){
-            this_event_category = event.category.map(ct => ct.replaceAll(" ", "-").toLowerCase()).join(" ");
-          };
-          if (typeof event.level !== 'undefined'){
-            this_event_level = event.level.replaceAll(" ", "-").toLowerCase();
-          };
-          eventsHTML += `<div class="col-md-6 col-lg-4 event" data-event-location="${this_event_local}"
-                          data-event-category="${this_event_category}" data-event-level="${this_event_level}">
+          eventsHTML += `<div class="col-md-6 col-lg-4 event" data-event-location="${event.location}"
+                          data-event-category="${event.category}" data-event-level="${event.level}">
```

Comment:
Everything is good 👍 only these lines can be just (since we don't use any filter)
```
<div class="col-md-6 col-lg-4">
```

Reply:
Thanks for reviewing and seeing that! I thought they belonged to the thumbnail part at first. I did a new commit and edited it accordingly :)